### PR TITLE
Add search widget

### LINKED ISSUE #1336: Composant recherche
Full text search
Limité à la recherche d'adresses.

### PR DESCRIPTION
**Pull request by tonio**
Fix #1336

### CODE REVIEW

**Comment by tonio** [comment]
Unless [CORS headers](https://github.com/Geoportail-Luxembourg/geoportailv3/issues/1364) are fixed, a CORS enable browser extension is needed to get it work.

**Comment by tonio** [comment]
Can now be tested without restrictions. ([CORS headers](https://github.com/Geoportail-Luxembourg/geoportailv3/issues/1364) are fixed)

File: jsapi/src/main.js [hunk 1]
```diff
@@ -335,6 +346,83 @@ lux.Map.prototype.addBgSelector = function(target) {
...
+    console.error('Search target should be a DOM Element or its id');
+    return;
+  }
+
+  var container = document.createElement('div');
+  container.classList.add('lux-search');
+
+  var input = document.createElement('input');
+  input.classList.add('lux-search-input');
+  input.setAttribute('placeholder', 'Search addresses');
```
Comment (pgiraud) [inline jsapi/src/main.js]: This should be translated.
We can address it later though.

**Comment by pgiraud** [comment]
Looks very good to me. Please merge.
We can address the translation issue later.

**Pull request merged**